base/etc/nginx/conf.d/location/10-block.conf

```diff
@@ -3,7 +3,7 @@ location ~ (^|/)\. {
 }
 
 # Protect files and directories from prying eyes.
-location ~* \.(log|engine|inc|install|make|module|profile|po|sh|.*sql|theme|twig|tpl(\.php)?|xtmpl|yml)(~|\.sw[op]|\.bak|\.orig|\.save)?$|/(\.(?!well-known).*|Entries.*|Repository|Root|Tag|Template|composer\.(json|lock)|web\.config)$|/#.*#$|\.php(~|\.sw[op]|\.bak|\.orig|\.save)$ {
+location ~ \.(log|engine|inc|install|make|module|profile|po|sh|.*sql|.*sql\.gz|theme|twig|tpl(\.php)?|xtmpl|yml)(~|\.sw[op]|\.bak|\.orig|\.save)?$|^\/(\.(?!well-known).*|Entries.*|Repository|Root|Tag|Template)$|(composer\.(json|lock)|web\.config|yarn\.lock|package(-lock)?\.json)$|^\/#.*#$|\.php(~|\.sw[op]|\.bak|\.orig|\.save)$ {
```

Comment:
`~*` means case insensitive matching while `~` is case sensitive. Imo it should be `~*` so we continue to block `.PHP` etc...

Also, is it worth breaking up this giant regex? It's really hard to figure out what's going on, evident from the bug 😅

Reply:
Giant regex comes from Drupal.

https://git.drupalcode.org/project/drupal/-/blob/11.x/.htaccess?ref_type=heads#L6

I would prefer we keep the string as close as possible so we can copy/paste to keep up to date. Looks like `log` is the outlier. @fubarhouse can you please confirm that and break out the things that aren't Drupal into their own rules. Also comment and link to Drupal core as the reference.

Drupal cores rule is case sensitive which is why we are seeing different behaviour on our Apache based clients (hosted on other solution) vs Skpr.

Reply:
So we're going to allow `.PHP` files?

Reply:
I've confirmed that `.PHP` files will not serve but will instead incur a `403` and provided a test for this.